src/styles/stars.scss

```diff
@@ -0,0 +1,26 @@
+.card__reviews {
+  display: flex;
+  justify-content: space-between;
+  font-size: 10px;
+  margin-bottom: 24px;
+  align-items: center;
+
+  &-stars {
+    display: flex;
```

Comment:
so that the component _Stars_ is independent, it is better to apply these styles for this selector `.stars`